crates/telemetry-subscribers/tests/reload.rs

```diff
@@ -0,0 +1,13 @@
+use telemetry_subscribers::TelemetryConfig;
+use tracing::{debug, info};
+use tracing_subscriber::EnvFilter;
+
+#[test]
+fn reload() {
+    let (_guard, reload_handle) = TelemetryConfig::new("test").with_env().init();
+
+    info!("Should be able to see this");
+    debug!("This won't be captured");
+    reload_handle.reload(EnvFilter::new("debug")).unwrap();
+    debug!("Now you can see this!");
+}
```

Comment:
Do you want to  [tracing_test](https://docs.rs/tracing-test/latest/tracing_test/) this?

Reply:
I dislike tracing_test, but i'll do you one better and just use our infra to write to a file and check the contents of it.

Reply:
Nice 😁